components/contributors.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import glamorous, {Div} from 'glamorous'
+import {Anchor} from '../components/styled-links'
+
+const Container = glamorous.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  '& > h3': {
+    margin: '20px',
+  },
+})
+
+const Img = glamorous.img({
+  height: 45,
+  width: 45,
+  borderRadius: '100%',
+  margin: '0 5px',
+  transition: 'transform .3s',
+  ':hover': {
+    transform: 'scale(1.2)',
+  },
+})
+
+const contributor = username =>
+  (<Anchor key={username} href={`https://github.com/${username}`}>
+    <Img src={`https://github.com/${username}.png?size=90`} />
+  </Anchor>)
+
+export default function Contributors({contributorsTitle, contributors = []}) {
```

Comment:
Got it working! :) I thought it would do some strange things (with content from the component and the content from the page). Just testing a bit more, commit is following!